arches/app/templates/base.htm

```diff
@@ -81,7 +81,7 @@
     {% include 'javascript.htm' %}
     
     {% if main_script %}
-        <script src="{% webpack_static '' %}/build/js/{{main_script}}.js"></script>
```

Comment:
Can you remove this change in favor of changing it on a project-by-project basis? There are several installations that would break if this change got commited.

Reply:
Unless I'm misunderstanding the purpose of the original this seems like a typo in the original that ends up only affecting Windows. 
The code is appending `"/build/js/{{main_script}}.js"` to the `webpack_static` variable. But the `webpack_static` variable is `"/static/"` therefor the combined string is `"/static//build/js/{{main_script}}.js"`. Note the double "//" between "static" and "build". 
On a default installation on either Windows or Linux you will see this code on pages that use `base.htm` (like /search):
```html
<script src="/static//build/js/views/search.js"></script>
``` 
Linux seems flexible enough to ignore it but Windows throws an error.

If there is a good reason the current functionality exists with the double slashes then I will have to add more code that does a modification only on Windows installs because this change would have to be done on every Windows install or none would be able to load any pages that use `base.htm`.

Reply:
ah I  can see your point here. The installations it would break do not have a trailing slash in the static_url. This is ok.